.github/workflows/mtags-auto-release.yml

```diff
@@ -0,0 +1,31 @@
+name: "Mtags auto release"
+on:
+  workflow_dispatch:
+    inputs:
+      scala_version:
```

Comment:
So, according to the [docs](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#manual-events) this has to be dispatched manually, am I right?

Reply:
Maybe in that case it is not needed? I think the workflow for releasing manually would be:
1. Checkout latest Metals tag
2. Add new tag with the new Scala version
3. Push it or push a tag with some fixes.

As for releasing for nightlies, we could just run it in the release job?

Reply:
> As for releasing for nightlies, we could just run it in the release job?

It will be required to merge in master to trigger it so it might be unclear when nightly support appears.

Actually, I wanted to use it for releasing other versions too. Just trigger this job, if there are problems, you will get a report, if everything is well - no actions required :steam_locomotive::railway_car::railway_car: